unit_test.py

```diff
@@ -1,4 +1,3 @@
-# This is file contains a unit test suit which completely test the backend
```

Comment:
lol, you left a sentence fragment on the next line, "functions of our app". It's okay to slow down.